app/models/miq_request_task.rb

```diff
@@ -85,6 +85,11 @@ def update_request_status
     update_and_notify_parent(:state => req_state, :status => req_status, :message => display_message(msg))
   end
 
+  def update_request_task(values)
+    update_attributes(:options => options.merge(values['options'] || {}))
+    self
```

Comment:
Why return `self`?

Reply:
@bdunne 
Thats the object that is being returned by the REST controller.

Reply:
Why?  That's not how ActiveRecord behaves.  I don't think the models should be concerned about the REST API.